### ISSUE TITLE
[BUG] <Offhand not seen>

### ISSUE THREAD

**Issue by SebSemos** [tr7zw/ItemSwapper#136]
### Version information

itemswapper-fabric-0.3.2-mc1.19.2

### Expected Behavior

Opening the item swapper menu on a block would let me select some stairs I have in offhand

### Actual Behavior

Items in off hand are red shaded and so nothing happen if we click on them

### Reproduction Steps

Put an item in off hand
Put an other item of the set in hotbar
Press hotkey
try to get the item of offhand

### Render engine

- [ ] Vanilla
- [X] Sodium(optionally with Iris)
- [ ] Optifine
- [ ] VulkanMod
- [ ] Canvas
- [ ] Other

### Additional information

Tested without any other mods

**Comment by tr7zw** [comment]
That is the exact intended behaviour. Items in the hotbar should not be pickable, since you already have them in the hotbar. There is a setting in the config screen to turn this off.

**Comment by SebSemos** [comment]
Yes, I should have said this was already turned off but items in offhand are still unpickable.

**Comment by tr7zw** [comment]
Hm, will have to check why it's not checking the offhand, but honestly, is this ever a wanted behaviour? You literally have the item in hand, so why pick it via the UI to your main hand? (Instead of just pressing F)

**Comment by SebSemos** [comment]
Might not be the most usefull feature, but I can't think of how many time I open the inventory to take an item I already had in either the hotbar or offhand 
This is why I noticed the issue in the first place... I was sure I had some stairs... and oh! took me about 30 secs de realize. 
Maybe the real issue are my eyes or my brain tho 😂😅
